sieve_of_Eratosthenes.py

```diff
@@ -0,0 +1,22 @@
+def sieve_of_eratosthenes(n):
+    
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
+++ b/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
@@ -1,5 +1,5 @@
 def sieve_of_eratosthenes(n):
-    
+
     prime = [True for i in range(n+1)]
     p = 2
     while (p * p <= n):
```